app/models/intake.rb

```diff
@@ -200,6 +200,9 @@ class Intake < ApplicationRecord
   belongs_to :vita_partner, optional: true
   belongs_to :triage_source, optional: true, polymorphic: true
 
+  attribute :updater
```

Comment:
There is a ticket in this sprint to allow volunteers to update the intake. When we do that, we can add an attribute of :updater in the controller with their user object, and when that is passed it will indicate to us that it is a _volunteer_ updating the intake instead of the client.

Another potential approach was to, again, to only update the last_responded_at once when the client completed the intake, and then never update it after with the assumption that all of those updates would be from volunteers. Open to refactoring.

Reply:
interesting! also relevant: this seems like it overlaps with the audit trail work, since we'll want to keep a record of who updates client information when. if we don't refactor now, we could do it later when that work happens (in the event that the way we implement the auditing conflicts with or is redundant with this).

this way is fine with me for now, but i also don't have the energy to think through a decision like this at the end of friday 🤣